spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -451,11 +486,104 @@ def get_spikes(
         result = numpy.column_stack((spike_ids, spike_times))
         return result[numpy.lexsort((spike_times, spike_ids))]
 
+    def get_rewires(
+            self, label, buffer_manager, placements,
+            application_vertex, variable, machine_time_step):
+        """ Read rewires mapped to time and neuron IDs from the SpiNNaker\
+            machine.
+
+        :param str label: vertex label
+        :param buffer_manager: the manager for buffered data
+        :type buffer_manager:
+            ~spinn_front_end_common.interface.buffer_management.BufferManager
+        :param ~pacman.model.placements.Placements placements:
+            the placements object
+        :param application_vertex:
+        :type application_vertex:
+            ~pacman.model.graphs.application.ApplicationVertex
+        :param str variable:
+        :param int machine_time_step: microseconds
+        :return:
+        :rtype: ~numpy.ndarray(tuple(int,int,int,int))
+        """
```

Comment:
If this is the way to get event variables out, it should probably take the variable name, and the protect against getting other types of variables through this function.

Reply:
I think I've guarded against this now by having _get_rewires(..) only called when get_events(variable, ..) with variable=REWIRING.